module/library/categories/helper.js

```diff
@@ -321,9 +394,65 @@ module.exports = class LibraryCategoriesHelper {
 	 * @returns {Object} category details
 	 */
 
-	static create(categoryData) {
+	static create(categoryData, files) {
 		return new Promise(async (resolve, reject) => {
 			try {
+				if (files && files.cover_image) {
+					let coverImages = files.cover_image
```

Comment:
@borkarsaish65 we support passing multiple files during category creation right!

Reply:
yes